code-challenges.test.js

```diff
@@ -14,47 +22,148 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibLength2 = 10
+describe("fibArray" , () => {
+  it("takes in a number greater than 2 and returns an array with the same # of characters in the Fibonacci sequence.", () => {
+    const fibLength1 = 6
+    const fibLength2 = 10
+    expect(fibArray(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    // expect(fibArray(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 37, 55,])
+  })
+})
+
+// Expected output: [1, 1, 2, 3, 5, 8]
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// RED - GOOD ERROR
+// FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
+//   ReferenceError: fibArray is not defined
+
+
 // b) Create the function that makes the test pass.
 
+
+  //Create a function named fibArray that takes in an inputVarible
```

Comment:
I personally like the pseudo code wrapped around your code because they also act as notes, but other instructors or future supervisors may not agree. Be open and flexible, but I love it! The details and the vocab is awesome!